v1.1/licensing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Licensing
+summary: Request and set trial and enterprise license keys for CockroachDB
+toc: false
+---
+
+This page shows you how to request and set the trial and enterprise license keys for CockroachDB.
+
+<div id="toc"></div>
+
+CockroachDB distributes a single binary that contains both core and [enterprise features](https://www.cockroachlabs.com/pricing/). You can use core features without any license key. However, to use the enterprise features, you need either a trial or an enterprise license key.
```

Comment:
I always find it strange to have a paragraph after the toc without a section heading. Let's make this the first paragraph on the page, followed by "This page shows you..." as the second paragraph.

Reply:
Fixed.